### DISCUSSION TITLE
Howto recover my fluxcd and gitlab

### DISCUSSION THREAD

**Discussion by nuxman** [fluxcd/flux2#2871] [Q&A] [answered]
Hello I did something yesterday by mistake that damaged my cluster, I installed my gitlab via kustomize and used with fluxcd, what happened is that I commented the gitlab of kustomization.yaml and did the commit and it came to delete all resources from the cluster, what I have on the server are the namespaces of flux and flux-system, my question is how can I recover my Gitlab to the same cluster after I did this ? I appreciate any help

**Comment by nuxman** [comment]
actually there is no gitlab url anymore because I commented out gitlab's kustomization.yaml and I have nothing left in the cluster, there is only flux and flux-system namespace left, there is no longer a my git server, so I don't know how to reverse this

**Comment by stefanprodan** [comment, marked as answer]
Run `flux uninstall`, this will delete all Flux components without touching any other workloads, and then rerun `flux bootstrap`.

**Reply by nuxman** [reply]
I just have to type flux boostrap? no other parameters needed? @stefanprodan

**Reply by nuxman** [reply]
And I still tried to apply the yaml I have in the repository I have left locally yesterday, will I harm this method you are sharing with me?

**Reply by stefanprodan** [reply]
You need to run `bootstrap` with the same params as before. There is no need to manually apply anything, bootstrap does it all. Docs here: https://fluxcd.io/docs/installation/
